src/main/java/org/jenkinsci/plugins/badge/BadgeActionFactory.java

```diff
@@ -16,22 +16,42 @@
 @Extension
 public class BadgeActionFactory extends TransientActionFactory<Job> {
 
+    /**
+     * TO DO
+     */
     private final ImageResolver iconResolver;
 
+    /**
```

Comment:
[Checkstyle] WARNING: First sentence of Javadoc is incomplete (period is missing) or not present.